### PR TITLE
Adding a simple VoC increase to Reff & setting a fixed end date to second wave

### PR DESCRIPTION
**Pull request by tobinsouth**
Early commits add at Beta(6,14)+1 increase to the Reff when a (new) VoC argument is passed. Expect this to change somewhat as new VoC are introduced and more complexity is needed.

A fixed date is set for the end of the second wave in NSW.

Some general house cleaning is done to remove data files from the git and add arguments to the bash scripts. 20000 sims are now run as too many simulations are being rejected in VIC.

### CODE REVIEW

File: analysis/cprs/generate_posterior.py [hunk 1]
```diff
@@ -303,11 +303,14 @@
     ##Second wave inputs
     sec_states=sorted(['NSW','VIC'])
     sec_start_date = '2020-06-01'
-    if data_date > pd.to_datetime("2020-06-12"):
-        possible_end_date = data_date - timedelta(10)#subtract 10 days to aovid right truncation
-    else:
-        possible_end_date = pd.to_datetime("2020-06-01")
-    sec_end_date = possible_end_date.strftime('%Y-%m-%d')
+    sec_end_date = '2021-01-18'
```
Comment (tdennisliu) [inline analysis/cprs/generate_posterior.py]: Just be mindful that this is a global end date on the second wave, and at line 348 there are state specific date ranges. Don't know what is a better solution, but might lead to conflicts later.

File: analysis/forecast_plots.py [hunk 2]
```diff
@@ -142,6 +144,14 @@ def read_in_Reff(forecast_R=None,R_I=None,file_date = "2020-04-01"):
    
 
             df_forecast = df_forecast.loc[df_forecast.type==forecast_R]
+
+            if VoC_flag != '':
+                # Here we apply the same beta(6,14)+1 scaling from VoC to the Reff data for plotting
+                # We do so but editing a slice of the data frame. Forgive me for my sins.
+                VoC_dates_to_apply_idx = df_forecast.index[pd.to_datetime(df_forecast.date, format='%Y-%m-%d') > pd.to_datetime(file_date)]
+                df_slice_after_VoC = df_forecast.iloc[VoC_dates_to_apply_idx, 8:]
+                df_forecast.iloc[VoC_dates_to_apply_idx , 8:] = df_slice_after_VoC*(beta.rvs(6,14, size = df_slice_after_VoC.shape) + 1)
```
Comment (tdennisliu) [inline analysis/forecast_plots.py:155]: Why are the 8th column? Might want to comment in that for future Tobin (or Tobin-replacement :) )

**Review by tdennisliu — APPROVED** [review]
Changes look good. Would recommend changing the README (and possibly adding comments to bash scripts) to reflect the additional optional argument for VoC flag. Josh will almost certainly ask when you need to handover about how to properly add it.

File: analysis/forecast_plots.py (hunk 2 above)
Comment (tobinsouth) [reply analysis/forecast_plots.py:155]: Comments added. Future Tobin sends his thanks :)

**Review by tdennisliu — APPROVED** [review]
All looks good, happy for you to merge. Might want to merge this into master soon before it gets too large and too many features get added at once.

**Pull request merged**